account/src/main/java/com/caldevsupplychain/account/service/AccountService.java

```diff
@@ -12,13 +12,17 @@
 
 	UserBean updateUser(UserBean userBean);
 
+	UserBean setPassCode(UserBean userBean);
```

Comment:
reuse `updateUser` for any user field update

Reply:
fix